Ghidra/Framework/Generic/src/main/java/generic/concurrent/ConcurrentQ.java

```diff
@@ -63,13 +63,13 @@
  * QItemListener<ITEM, RESULT> itemListener = new QItemListener<ITEM, RESULT>() {
  *     public void itemProcessed(QResult<ITEM, RESULT> result) {
  *         RESULT result = result.getResult();}
- *             <font color="blue"><b>// work on my result...</b></font>{@code
+ *             }<span style="color:blue"><b>// work on my result...</b></span>
  *         }
  * };
  * 
- * ConcurrentQBuilder<ITEM, RESULT> builder = new ConcurrentQBuilder<ITEM, RESULT>();
+ * {@literal ConcurrentQBuilder<ITEM, RESULT> builder = new ConcurrentQBuilder<ITEM, RESULT>();
  * builder.setThreadPoolName("Thread Pool Name");
- * }<font color="blue"><b>builder.setListener(itemListener);</b></font>{@code
+ * }<span style="color:blue"><b>builder.setListener(itemListener);</b></span>{@code
```

Comment:
These new spans need line breaks.

Reply:
I noticed that the `{@literal }` tags and the curly braces from the code were conflicting. This caused one of them to be interpreted literally. Was this the reason for the line breaks? With the soon incoming correction it appears identical to the docs released with 9.1.